packages/expo-secure-store/CHANGELOG.md

```diff
@@ -8,6 +8,12 @@
 
 ### 🐛 Bug fixes
 
+## 15.0.7 — 2025-10-29
+
+### 💡 Others
+
+- [docs] Clarify `SecureStore` data persistence behavior in the documentation. ([#40729](https://github.com/expo/expo/pull/#40729) by [@dileepapeiris](https://github.com/dileepapeiris)) ([#40729](https://github.com/expo/expo/pull/40729) by [@dileepapeiris](https://github.com/dileepapeiris))
+
```

Comment:
```suggestion
```

Typically we wouldn't mark this in the library changelog as the library code has not changed, and the deployment of the docs is separate. 

If you decide to move the simulator comment as a note of the `requireAuthentication` prop you will have to edit `SecureStore.ts` which is a part of the code of the library. In that case the changelog comment will be necessary.